src/register/Register.jsx

```diff
@@ -0,0 +1,272 @@
+import React, { useState } from "react";
+import "./register.css";
```

Comment:
The naming convention is Register.jsx and Register.css

Reply:
just changed it now 👍